liquidator/src/one-inch.js

```diff
@@ -1,38 +1,42 @@
 const Token = artifacts.require("ExpandedERC20");
 const OneSplit = artifacts.require("OneSplit");
 
-const oneSplitAddress = "0xC586BeF4a0992C495Cf22e1aeEE4E446CECDee0E";
-
 // As defined in 1inch
 const ETH_ADDRESS = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
```

Comment:
This is an address defined in 1inch?

Reply:
Its [either address(0) or "0xEee...."](https://github.com/CryptoManiacsZone/1inchProtocol#methods)